prog/weaveproxy/main.go

```diff
@@ -10,21 +10,33 @@ import (
 )
 
 var (
-	version       = "(unreleased version)"
-	defaultTarget = "unix:///var/run/docker.sock"
-	defaultListen = ":12375"
+	version           = "(unreleased version)"
+	defaultDockerAddr = "unix:///var/run/docker.sock"
+	defaultListenAddr = ":12375"
 )
 
 func main() {
-	var target, listen string
-	var withDNS, withIPAM, debug, justVersion bool
-
+	var (
+		debug       bool
+		justVersion bool
+		c           = proxy.Config{
+			DockerAddr: defaultDockerAddr,
+			ListenAddr: defaultListenAddr,
+		}
```

Comment:
Exactly. That is just a usage string. If the option is not set, it doesn't modify the value of ListenAddr.